http/rest-client-reactive/src/main/resources/application.properties

```diff
@@ -8,3 +8,6 @@ quarkus.rest-client.encoder-mode-rfc1738.url=${vertx-server-url}
 quarkus.rest-client.encoder-mode-rfc1738.multipart-post-encoder-mode=RFC1738
 quarkus.rest-client.encoder-mode-rfc3986.url=${vertx-server-url}
 quarkus.rest-client.encoder-mode-rfc3986.multipart-post-encoder-mode=RFC3986
+
+# Set the Client endpoint to a non-existing domain to trigger a fault
```

Comment:
`to trigger a fault` I don't know that specifically that would mean here. either you mean failure or fault tolerance

Reply:
on second thought, I think fault is also acceptable.